helper/dockertest/benchmark.go

```diff
@@ -0,0 +1,56 @@
+package dockertest
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/docker/docker/api/types/container"
+	dockhelper "github.com/hashicorp/vault/sdk/helper/docker"
+)
+
+func CreateVaultBenchmarkContainer(t *testing.T, networkName string, vaultAddr string, vaultToken string) int64 {
```

Comment:
Originally I was returning the cleanup function, but because all the container does is run `vault-benchmark run` and then exits it doesn't need to be cleaned up.  I can return the two channels. Would that work better?

Reply:
I separated these out, lmk if that makes things more clear!